res/res_pjsip_header_funcs.c

```diff
@@ -52,13 +52,16 @@
 				<enumlist>
 					<enum name="read"><para>Returns instance <replaceable>number</replaceable>
 					of header <replaceable>name</replaceable>. A <literal>*</literal>
-					may be appended to <replaceable>name</replaceable> to iterate over all 
+					may be appended to <replaceable>name</replaceable> to iterate over all
 					headers <emphasis>beginning with</emphasis> <replaceable>name</replaceable>.
 					</para></enum>
 
 					<enum name="add"><para>Adds a new header <replaceable>name</replaceable>
 					to this session.</para></enum>
 
+					<enum name="pass"><para>Adds a new header <replaceable>name</replaceable>
```

Comment:
inherit, not pass, and document that this does NOT automatically pass through received SIP headers